File: CAPITULO 02/Java/Ex06.java
import java.io.*;
import java.util.*;
public class Ex06{
	public static void main(String[]args){
		int num,div;
		Scanner dado = new Scanner(System.in);
		System.out.println("Digite um numero ");
		num = dado.nextInt();
		div = num %2;
		if(div == 0)
			System.out.println("Numero par");
		else
			System.out.println("Numero impar");
	}
}

File: CAPITULO 02/C++/ex06.c
#include<stdio.h>
int main(){
    int num,div;
    printf("Digite um numero ");
    scanf("%d%*c",&num);
    div = num%2;
    if (div == 0)
        printf("Numero par");
    else
        printf("Numero impar");
    return 0;
}
